packages/app/src/components/DonationDialog.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <q-dialog v-model="inDonation">
+      <q-card class="donation-card">
+        <q-card-section>
+          <h6>Thank you for donating! 💕</h6>
+          <span>Your donations help keep our work alive.</span>
+          <span><br>All proceeds go to Grey Software.</span>
+        </q-card-section>
+        <q-card-section>
+          <div class="column items-center">
+            <q-btn
```

Comment:
Please add the ``rounded`` prop to all buttons to keep them more consistent with the rounded buttons in the UI :)

Reply:
done.